packages/fluent/lib/swatches/fluent-main.json

```diff
@@ -7,10 +7,10 @@
     "api": "modern",
     "previewColors": [
         "#ffffff",
-        "#f5f5f5",
-        "#ff6358",
-        "#d6534a",
-        "#424242"
+        "#faf9f8",
+        "#0078d4",
+        "#005a9e",
```

Comment:
The color here should be secondary? - #979593
https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/core/color-system/_palettes.scss#L13